snippets/javascript/nuxt/nuxt.json

```diff
@@ -0,0 +1,87 @@
+{
+  "nuxt-nuxtlink": {
+    "prefix": ["nuxt nuxtlink", "nuxtlink"],
+    "body": ["<nuxt-link $1>$2</nuxt-link>$0"],
+    "description": "nuxt-link element",
+    "scope": "vue-html"
+  },
+  "nuxt-nuxtlinkto": {
+    "prefix": ["nuxt nuxtlinkto", "nuxtlinkto"],
+    "body": ["<nuxt-link to=\"$1\">$2</nuxt-link>$0"],
+    "description": "<nuxt-link to=''></nuxt-link>. nuxt-link element",
+    "scope": "vue-html"
+  },
+  "nuxt-nuxtnamedlinkto": {
+    "prefix": ["nuxt nuxtnamedlinkto", "nuxtnamedlinkto"],
+    "body": [
+      "<nuxt-link :to=\"{\t${1:key}: '${2:value}', ${3:key}: {\t${4:key}: '${5:value}'\t}\t}\">$6</nuxt-link>$0"
+    ],
+    "description": "<nuxt-link :to='{name: ''}'></nuxt-link>. nuxt-link element",
+    "scope": "vue-html"
+  },
+  "nuxt-nuxtview": {
+    "prefix": ["nuxt nuxtview", "nuxtview"],
+    "body": ["<nuxt>$1</nuxt>$0"],
+    "description": "nuxt-view element",
+    "scope": "vue-html"
+  },
+  "nuxt-nuxtnamedview": {
+    "prefix": ["nuxt nuxtnamedview", "nuxtnamedview"],
+    "body": ["<nuxt name=\"$1\">$2</nuxt>$0"],
+    "description": "nuxt-named-view element",
+    "scope": "vue-html"
+  },
+  "nuxt-nuxtchild": {
+    "prefix": ["nuxt nuxtchild", "nuxtchild"],
+    "body": ["<nuxt-child $1/>$0"],
+    "description": "nuxt-child element",
+    "scope": "vue-html"
+  },
+  "nuxt-async-data": {
+    "prefix": ["nuxt asyncdata", "vasyncdata"],
+    "body": [
+      "asyncData ({isDev, route, store, env, params, query, req, res, redirect, error}) {",
+      "\n},"
+    ],
+    "description": "Fetch data and pre-render it on the server without using a store. The result from asyncData will be merged with data",
+    "scope": "vue"
+  },
+  "nuxt-head": {
+    "prefix": ["nuxt head", "vhead"],
+    "body": [
+      "head () {",
+      "\treturn {",
+      "\t\ttitle: '',",
+      "\t\tmeta: [",
```

Comment:
I guess there is a syntax error:
```
head () {
  return {
    title: '',
    meta: [
      { hid: 'description', name: 'description', content: 'My custom description' }
  }
},
```
If meta is an array there should be a closing bracket.